packages/core/src/types/oas3.ts

```diff
@@ -233,7 +233,7 @@ const MediaType: NodeType = {
 
 const Example: NodeType = {
```

Comment:
And the example/examples are different in OAS 3: https://spec.openapis.org/oas/v3.2.0.html#media-type-object.
Maybe we have to adjust the node types to reflect that first?